src/utils/api/stream.js

```diff
@@ -187,15 +188,18 @@ export default class Stream {
 
             const link = "/thread/" + message.conversation_id;
 
-            const notification = new Notification(title, {
-                icon: '/static/images/android-desktop.png',
-                body: snippet
+            const notification = new Notify(title, {
+                icon: require('@/../public/images/android-desktop.png'), // Require to resolve
+                body: snippet,
+                notifyClick: onclick
             });
 
-            notification.onclick = () => {
+            const onclick = () => {
                 window.focus();
                 router.push(link).catch(() => {});
             };
```

Comment:
Looks like clicks weren't working on the notification, since the action was defined after the notification. I fixed it

https://github.com/klinker-apps/pulse-sms-web/commit/5c765ab38ad762966ebf251d5bd6bef7d34597a5